### PR TITLE
Fix some windows warnings

### PR DESCRIPTION
**Pull request by epernod**
Just a small PR to remove some warnings on windows and say hello to this project!

### CODE REVIEW

**Comment by VannesteFelix** [comment]
Thank you for this PR @epernod !

**Comment by VannesteFelix** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
[ci-build]

**Comment by marques-bruno** [comment]
Yo @epernod  because we have issue with the CI, I'll merge it in a [dev] branch temporarily, so that I can build it with a JK job. If it passes, we'll merge it back in master ;)

File: src/component/forcefield/HyperReducedHelper.h [hunk 1]
```diff
@@ -173,7 +173,7 @@ class HyperReducedHelper : public virtual core::objectmodel::BaseObject
     void saveGieFile(unsigned int nbElements){
         if (d_prepareECSW.getValue())
         {
-            unsigned int numTest = this->getContext()->getTime()/this->getContext()->getDt();
+            unsigned int numTest = unsigned int(this->getContext()->getTime()/this->getContext()->getDt());
```
Comment (marques-bruno) [inline src/component/forcefield/HyperReducedHelper.h]: This does not compile with GCC, as it thinks that it's supposed to call the ctor for int() and that "unsigned" is just a dangling keyword...
to replace with calls to unsigned directly, or size_t, uint32_t etc.

**Pull request merged**